dev/tests/integration/testsuite/Magento/CatalogInventory/Model/StockItemSave/StockItemDataChecker.php

```diff
@@ -166,21 +178,24 @@ private function assertArrayContains(array $expected, array $actual)
      */
     private function checkIntegrityWithInventory(Product $product, array $expectedData)
     {
-        $searchCriteria = $this->searchCriteriaBuilder
-            ->addFilter(SourceItemInterface::SOURCE_CODE, $this->defaultSourceProvider->getCode())
-            ->addFilter(SourceItemInterface::SKU, $product->getSku())
-            ->create();
-        $sourceItems = $this->sourceItemRepository->getList($searchCriteria)->getItems();
-        Assert::assertCount(1, $sourceItems);
-
-        $sourceItem = reset($sourceItems);
-        Assert::assertEquals(
-            $expectedData[StockItemInterface::QTY],
-            $sourceItem->getQuantity()
-        );
-        Assert::assertEquals(
-            $expectedData[StockItemInterface::IS_IN_STOCK],
-            (int)$sourceItem->getStatus()
-        );
+        // soft dependency in tests because we don't have possibility replace tests from different modules
```

Comment:
Need to move this check inti `checkStockItemData` and wrap `$this->checkIntegrityWithInventory($product, $expectedData);` call

Reply:
Done.